tests/test_integration_org_unit.py

```diff
@@ -397,6 +397,63 @@ def test_create_org_unit(self, m):
             },
         )
 
+    def test_create_org_unit_fails_validation_outside_org_unit(self):
+        """Validation should fail when date range is outside of org unit
+        range """
+        self.load_sample_structures()
+
+        c = lora.Connector(virkningfra='-infinity', virkningtil='infinity')
+
+        payload = {
+            "name": "Fake Corp",
+            "parent": {
+                'uuid': "2874e1dc-85e6-4269-823a-e1125484dfd3"
+            },
+            "org_unit_type": {
+                'uuid': "ca76a441-6226-404f-88a9-31e02e420e52"
+            },
+            "addresses": [
+                {
+                    "address_type": {
+                        "example": "20304060",
+                        "name": "Telefonnummer",
+                        "scope": "PHONE",
+                        "user_key": "Telefon",
+                        "uuid": "1d1d3711-5af4-4084-99b3-df2b8752fdec",
+                    },
+                    "value": "11 22 33 44",
+                },
+                {
+                    "address_type": {
+                        "example": "<UUID>",
+                        "name": "Adresse",
+                        "scope": "DAR",
+                        "user_key": "Adresse",
+                        "uuid": "4e337d8e-1fd2-4449-8110-e0c8a22958ed"
+                    },
+                    "uuid": "44c532e1-f617-4174-b144-d37ce9fda2bd",
+                },
+            ],
+            "validity": {
+                "from": "2010-02-04T00:00:00+01",
+                "to": "2017-10-22T00:00:00+02",
+            }
+        }
+
+        expected = {
+            'description': 'Date range exceeds validity '
+                           'range of associated org unit.',
+            'error': True,
+            'error_key': 'V_DATE_OUTSIDE_ORG_UNIT_RANGE',
+            'org_unit_uuid': '2874e1dc-85e6-4269-823a-e1125484dfd3',
+            'status': 400,
+            'valid_from': 'Thu, 04 Feb 2010 00:00:00 GMT',
```

Comment:
This also isn't related to this change, strictly speaking, but we should fix this to use ISO 8601 format just like the rest of our API.